pages/common/awk.md

```diff
@@ -21,3 +21,7 @@
 - Sum the values in the first column and pretty-print the values and then the total:
 
 `awk '{s+=$1; print $1} END {print "--------"; print s}' {{filename}}`
+
+- Print every third line starting from the first line
```

Comment:
As mentioned by the bot, this just needs to end in a colon. Other than this, I can't see anything wrong :+1:

Reply:
Sure!